docs/source/_themes/sphinx_rtd_theme/static/css/theme.css

```diff
@@ -3488,7 +3488,7 @@ a.wy-text-neutral:hover {
   color: #595959 !important;
 }
 
-h1, h2, h3, h4, h5, h6, legend {
+h1, h2, h3, h4, h5, h6, .rst-content .toctree-wrapper p.caption, legend {
```

Comment:
This looks like reset styles, they suppose to make browser implementations for particular elements look the same. I'd rather you copy font's weight and family to specific selector you already have down below. Not a deal breaker, but slightly more preferred way.